test/unit/api/ws/rpc/disconnect.js

```diff
@@ -44,7 +44,7 @@ describe('disconnect', () => {
 		it('should call peer.socket.disconnect', () => {
 			disconnect(validPeer);
 			// The socket should be deleted.
-			expect(validPeer.socket, undefined);
+			expect(validPeer.socket.destroy).calledOnce;
```

Comment:
if the change should be equal, it should be
`expect(validPeer.socket).to.be.undefined`

Reply:
It doesn't work this way. I tried before opening the PR. Do you have any other idea? Actually I had some problems trying to understand this one.

Reply:
by looking at the code, i guess original test was ambiguous i guess.
i think what you wrote `expect(validPeer.socket.destroy).calledOnce;` is ok, but prefer `expect(validPeer.socket.destroy).to.be.calledWithExactlyWith(1000, 'Intentionally disconnected from peer because of disconnect call');`